### PR TITLE
[ES] Add content/es/docs/concepts/overview/working-with-objects/finalizers

### PR DESCRIPTION
**Pull request by marianogg9**
Added:
- glossary entry `content/es/docs/reference/glossary/finalizer.md`.
- [ES] translation `content/es/docs/concepts/overview/working-with-objects/finalizers.md`.

### CODE REVIEW

**Comment by utkarsh-singh1** [comment]
Hi @marianogg9, Sign EasyCla first,

please follow these guides to sign easycla,

You can find about EasyCLA here - https://docs.linuxfoundation.org/lfx/easycla

and the steps to sign EasyCLA are here - https://docs.linuxfoundation.org/v2/easycla/contributors/individual-contributor

You can start your signing EasyCLA from here - https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/38056/#/?version=2

**Comment by marianogg9** [comment]
> Hi @marianogg9, Sign EasyCla first,
> 
> please follow these guides to sign easycla,
> 
> You can find about EasyCLA here - https://docs.linuxfoundation.org/lfx/easycla
> 
> and the steps to sign EasyCLA are here - https://docs.linuxfoundation.org/v2/easycla/contributors/individual-contributor
> 
> You can start your signing EasyCLA from here - https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/38056/#/?version=2

@utkarsh-singh1 done

**Review by Arhell — COMMENTED** [review]
/easycla

File: content/es/docs/concepts/overview/working-with-objects/finalizers.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Un ejemplo usual de un finalizador es `kubernetes.io/pv-protection`, el cual
+previene el borrado accidental de objetos `PersistentVolume`. Cuando un objeto
+`PersistentVolume` está en uso por un Pod, Kubernetes agrega el finalizador
+`pv-protection`. Si intentas elimiar el `PersistentVolume`, este pasa a un estado
+`Terminating`, pero el controlador no puede eliminarlo ya que existe el finalizador.
+Cuando el Pod deja de utilizar el `PersistentVolume`, Kubernetes borra el finalizador
+`pv-protection` y el controlador borra el volumen.
+
+## Referencias de dueño, etiquetas y finalizadores (#dueños-etiquetas-finalizadores)
```
Comment (krol3) [inline content/es/docs/concepts/overview/working-with-objects/finalizers.md:49]:
```suggestion
## Referencias de dueño, etiquetas y finalizadores {#owners-labels-finalizers}
```

**Review by krol3 — CHANGES_REQUESTED** [review]

**Review by krol3 — APPROVED** [review]
/lgtm

**Pull request merged**